README.md

```diff
@@ -8,16 +8,30 @@ The public facing website for BrickHack.
 
 # Getting Started
```

Comment:
If you want to go for maximum Cool README I'd suggest putting a screenshot of the preregister site right at the top! See the [brickhack-mobile-ios readme](https://github.com/codeRIT/brickhack-mobile-ios/) for how I centered that image if you'd like.

Reply:
I feel like that might be a little less relevant for this since it's a website you can easily visit.